.azure/ipu-tests.yml

```diff
@@ -18,6 +18,7 @@ pr:
       - ".actions/**"
       - ".azure/ipu-tests.yml"
       - "requirements/fabric/**"
+      - "src/lightning/*"
```

Comment:
Can we be explicit with what we are adding here?

Also these changes need to be reflected in the checkgroup config too

Reply:
added :)